### ISSUE TITLE
Type comments are positioned incorrect

### ISSUE THREAD

**Issue by rioj7** [asweigart/pyinputplus#12]
For the following functions the type comment line is placed incorrect

```
line 267: def inputCustom()
line 330: def inputNum()
```

The type recognition is incorrect because the type line is placed incorrect

```
def inputCustom(
    # type: (Callable, str, .....) -> Any
    customValidationFunc,
    prompt="",
    .....):
    """Prompts the user to enter input. ....."""
```

it should be AFTER the argument list.

```
def inputCustom(
    customValidationFunc, prompt="", .....):
    # type: (Callable, str, .....) -> Any
    """Prompts the user to enter input. ....."""
```

**Comment by asweigart** [comment]
Thanks! Fixed in 0.2.11.

**Comment by rioj7** [comment]
Thanks.

For the record the fix is in commit e099441.

To automatically get the issue closed and the commit mentioned in the issue you can add to the commit message things like: `fix #12` or `close #12`
